gqlalchemy/utilities.py

```diff
@@ -13,9 +13,15 @@
 # limitations under the License.
 
 from abc import ABC, abstractmethod
+import inspect
 import math
+
 import numpy as np
-import torch
+
+try:
+    import torch
+except ModuleNotFoundError:
+    torch = None
 
 from datetime import datetime, date, time, timedelta
```

Comment:
`datetime`, `enum` and `Typing` import should be above `numpy`.
I know you didn't put it here, but please update it. The dependencies should always follow this order:
1. standard library imports
2. related third-party imports
3. Local application/library-specific imports

- there should be a blank line between each group of imports.

Reply:
Done